docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -0,0 +1,179 @@
+|`count`|Integer|The number of users that need to match the filter expression to satisfy the policy|
+
+#### Example
+
+The policy below specifies that the `prod-access` role
+must have a minimum of two users with the role `auditor` and the mode `moderator` present in the session
+to start it. The policy applies to SSH and Kubernetes sessions only.
+
+When a user with this require policy starts a session, it will be pending
+until the policy is fulfilled. 
+
+```yaml
+kind: role
+metadata:
+  name: prod-access
+spec:
+  allow:
+    require_session_join:
+      - name: Auditor oversight
+        filter: 'contains(user.roles, "auditor")'
+        kinds: ['k8s', 'ssh']
+        modes: ['moderator']
+        count: 2
+```
+
+### `join_sessions`
+
+#### Options
+
+The following are required options for `join_sessions`:
+
+|Option|Type|Description|
+|---|---|---|
+|`name`|String|The name of the allow policy|
+|`roles`|[]String|A list of names for Teleport roles that this policy applies to. Users with this role are eligible to join a Moderated Session under this policy.|
+|`kinds`|`[]`[Session kind](./moderated-sessions.mdx#session-kinds)|The kind of session that the policy applies to|
+|`modes`|`[]`[Participant mode](./moderated-sessions.mdx#participant-modes)|The participant mode that paplies to the user joining the Moderated Session under this policy|
+
+#### Example
+
+The following allow policy attaches to the role `auditor` and allows one to join
+SSH and Kubernetes sessions started by a user with the role `prod-access` as a moderator or observer.
+
+```yaml
+kind: role
+metadata:
+  name: auditor
+spec:
+  allow:
+    join_sessions:
+      - name: Auditor oversight
+        roles : ['prod-access']
+        kinds: ['k8s', 'ssh']
+        modes: ['moderator', 'observer']
+```
+
+### Filters
+
+A filter expression is a string statement used to define logic based on a set of input variables.
+The filter expressions follow a restricted subset of Go syntax and have the environment supports
+the following functions and operators:
+- `contains(set, item)`: Returns true if the item is in the set, otherwise false. The set can be a list or an array
+- `equals(a, b)`: Returns true if the two values are equal, otherwise returns false.
+- `![expr]`: Negates a boolean expression.
+- `[expr] && [expr]`: Performs a logical AND on two boolean expressions.
+- `[expr] || [expr]`: Performs a logical OR on two boolean expressions.
+
+Filter expressions allow for more detailed control over the scope of an allow policy or require policy.
+`equals(user.name, "adam") || contains(user.roles, "cs-observe")` is a simple example of a filter expression
+that evaluates to true if the user is Adam or if the user has the trait `cs-observe`.
+
+Require policies can specify which users they consider as valid with a filter expression.
```

Comment:
Should we move this and the next sentence to the relevant table row in the section on configuring require policies?